ponyo/cca_core.py

```diff
@@ -159,7 +159,7 @@ def compute_ccas(sigma_xx, sigma_xy, sigma_yx, sigma_yy, verbose=True):
     try:
         ux, sx, vx = np.linalg.svd(arr_x_stable)
         uy, sy, vy = np.linalg.svd(arr_y_stable)
-    except:
+    except BaseException:
```

Comment:
Is there anything more specific that you expect to get here? Also, if this isn't your code, you could add ignoring it to your flake config and revert this change if you want.

Reply:
Good point. This script was imported and so I will revert the changes and add the file to the setup.cfg